shell/config/table-headers.js

```diff
@@ -948,6 +948,16 @@ export const EXPIRES = {
   formatter: 'LiveExpiryDate'
 };
 
+export const LAST_USED = {
+  name:      'lastUsed',
+  value:     'lastUsedAt',
+  labelKey:  'tableHeaders.lastUsed',
+  align:     'left',
+  sort:      ['lastUsedAt'],
+  width:     200,
+  formatter: 'LiveExpiryDate'
```

Comment:
Tricky one. Do existing tokens get updated on upgrade? when lastUsedAt is empty we'll show `Never`, which is a bit inaccurate.

![image](https://github.com/user-attachments/assets/b48f093e-3ea4-4651-a2fe-a42e546f696b)

Could add something like `formatterOpts: { missingKey: 'generic.unknown' },` and plumb that into LifeExpiryDate via the prop below?

 ```   
 missingKey: {
      type:    String,
      default: `generic.never`,
    },
```

Reply:
Will take a look - agreed

Reply:
.. although for a newly created token, I think it is empty, so showing Never is accurate, since it's never been used... will check on a fresh system by creating a token then upgrading Rancher.

Reply:
ah good point. maybe just leaving empty would be the best option?